pyuvdata/uvcal/uvcal.py

```diff
@@ -561,6 +563,45 @@ def __init__(self):
 
         super(UVCal, self).__init__()
 
+    @staticmethod
+    def new(**kwargs):
+        """
+        Create a new UVCal object.
+
+        All parameters are passed through to the
+        :func:`~pyuvdata.uvcal.initializers.new_uvcal` function.
+        """
```

Comment:
I'd prefer to do the same thing you did in the UVData function so that the docs will show all the options here.